babel.config.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-undef
 module.exports = {
-  presets: [['@babel/preset-env', { targets: { node: 'current' } }]]
-};
\ No newline at end of file
+  presets: [ ['@babel/preset-env', { targets: { node: 'current' } }] ]
```

Comment:
This line is added in **Add data parser and test** any chance this chance could be made there? instead of added one way and then changed in the next patch in the series?